scripts/deploy.sh

```diff
@@ -25,6 +25,9 @@ else
   echo "Snapshot deployed!"
 fi
 
+
+cat README.md | grep -v 'the official docs' > docs/source/essentials/get-started.md
```

Comment:
Wondering if we could just not have a `get-started.md` file and just have this script here. It would essentially create the `get-started.md` file when it gets clobbered with the `cat README.md`

Reply:
Not sure I follow. The original intent was to remove `get-started.md` from source control since it duplicates a lot of information with `README.md`

Reply:
> if we delete the get-started.md from source control, all we'd have to do is run this cat README.md | grep -v 'the official docs' > docs/source/essentials/get-started.md in order for it to actually show up when it's deployed on gatsby.

Exactly. This avoids duplicating the file in source control. You will need to do that from netlify.yml though as `deploy.sh` might not be recognized by neetlify/gatsby